### DISCUSSION TITLE
How to use multi GPUs in dp train during dpgen process

### DISCUSSION THREAD

**Discussion by ZSHtju** [deepmodeling/dpdispatcher#291] [Q&A] [answered]
Dear developers,
I have one question regarding dpgen.
Currently, I have several GPUs in my local workstation, and I want to fully use them during dpgen.
For instance, we will train four DNNs simultaneously. How can I respectively use one GPU for one DNN?
BR//Shenghui

**Comment by wanghan-iapcm** [comment]
I transfer this question to the discussion of [dpdispatcher](https://github.com/deepmodeling/dpdispatcher/)

**Comment by njzjz** [comment, marked as answer]
Please read documentation [Example: Running multiple MD tasks on a GPU workstation](https://docs.deepmodeling.com/projects/dpdispatcher/en/latest/examples/shell.html#running-multiple-md-tasks-on-a-gpu-workstation).

**Reply by ZSHtju** [reply]
Thanks a lot!

**Reply by llsnnn** [reply]
> Thanks a lot!

Have you successfully used this method? I still cannot make the four graphics cards work according to what I wrote in that article.
